api/src/team/application/membership/membership.controller.js

```diff
@@ -59,12 +59,25 @@ const findPaginatedFilteredMemberships = async function (request) {
   return membershipSerializer.serialize(memberships, pagination);
 };
 
+async function updateLastAccessedAt(request, h, dependencies = { requestResponseUtils }) {
+  const userId = dependencies.requestResponseUtils.extractUserIdFromRequest(request);
+  const organizationId = request.params.organizationId || request.params.id;
```

Comment:
La route étant `/organizations/{organizationId}/me` pourquoi gérer un cas `request.params.id` ? Peut-être y-a-t-il quelque chose qui m'échappe ?

```suggestion
  const organizationId = request.params.organizationId;
```